articles/azure-functions/functions-monitoring.md

```diff
@@ -586,7 +586,7 @@ The `tagOverrides` parameter sets the `operation_Id` to the function's invocatio
 
 ## Dependencies
 
-Functions v2 automatically collects dependencies for HTTP requests, ServiceBus, and SQL.
+Functions v2 automatically collects dependencies for HTTP requests, ServiceBus, EventHub, and SQL.
```

Comment:
Basing it on "Automatic collection of" section of https://docs.microsoft.com/en-us/azure/azure-monitor/app/azure-functions-supported-features